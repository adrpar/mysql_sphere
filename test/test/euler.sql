SET sphere_outprec=8;

-- checking Euler transformation operators

SELECT sequal(strans('-10d,0d,10d,ZZZ'), strans('-10d,0d,10d,XXX'));
SELECT snotequal(strans('-40d,0d,40d,ZZZ'), strans('-40d,0d,40d,XXX'));

