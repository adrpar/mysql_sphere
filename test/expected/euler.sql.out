sequal(strans('-10d,0d,10d,ZZZ'), strans('-10d,0d,10d,XXX'))
1
snotequal(strans('-40d,0d,40d,ZZZ'), strans('-40d,0d,40d,XXX'))
0
